src/map/serde_seq.rs

```diff
@@ -35,8 +33,6 @@ use crate::IndexMap;
 ///
 /// This behaves like [`crate::serde_seq`] for `IndexMap`, serializing a sequence
```

Comment:
We can tackle that in a separate PR if you want to make the internal change? I do agree it would make sense to move the `Slice` impls. If we move to `map/serde.rs` and `set/serde.rs` though, to keep it consistent it might also be worth moving `rayon/map.rs` to `map/rayon.rs` and `rayon/set.rs` to `set/rayon.rs`.